cpg-library/build.gradle.kts

```diff
@@ -136,6 +136,7 @@ dependencies {
 
     // CDT
     api("org.eclipse.cdt:core:7.2.0.202102251239")
+    api("org.neo4j:neo4j-ogm-bolt-driver:3.2.10")
```

Comment:
Why was this dependency added here?

Reply:
Unintentional, I need that for testing will remove.